low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -25,7 +25,8 @@
 
 from framework.base.internal_msgQ import InternalMsgQ
 from framework.base.module_thread import ScheduledModuleThread
-from framework.base.sspl_constants import enabled_products, DATA_PATH

```

Comment:
not crossing 80 char length, why changing then

Reply:
reverted the change